Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -190,17 +187,20 @@ static Color GetAccentColor()
 
 		class AndroidDeviceInfo : DeviceInfo
 		{
-			readonly IDeviceInfoProvider _formsActivity;
+			readonly Context _formsActivity;
 			readonly Size _pixelScreenSize;
 			readonly double _scalingFactor;
 
 			Orientation _previousOrientation = Orientation.Undefined;
 
-			public AndroidDeviceInfo(IDeviceInfoProvider formsActivity)
+			public AndroidDeviceInfo(Context formsActivity)
```

Comment:
There's no guarantee it is an `Activity`. The information we need exists on `Context` and we are guaranteed to have a `Context`, so we should take that information.

If we are lucky and have an `Activity` then we can definitely cast to that and get more info, or if it's an IDeviceInfoProvider we could cast to that. However we cannot rely on that always being true.